database/migration/changesets/20200315_CreateWorkloadTable.xml

```diff
@@ -0,0 +1,123 @@
+                             to get the value
+                             in the 'table' column below.">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="commit" type="text"
+                    remarks="The Git commit hash of WorkFlowLauncher
+                             that managed this workload.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="timestamptz"
+                    defaultValueComputed="now()"
+                    remarks="When was the workload row created here?">
+                <constraints nullable="false"/>
+            </column>
+            <column name="creator" type="text"
+                    remarks="Who created this workload?">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cromwell" type="text"
+                    remarks="The URL of the Cromwell instance
+                             running the workflows.
+                             Should be a common prefix
+                             of the 'cromwell' column
+                             in the 'table' table.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="finished" type="timestamptz"
+                    remarks="When was the workload finished?
+                             All workflows either succeeded
+                             or were abandoned by this time.">
+            </column>
+            <column name="input" type="text"
+                    remarks="Common prefix
+                             of the input URLs
+                             in 'table'.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="output" type="text"
+                    remarks="Common prefix
+                             of the output URLs
+                             in 'table'.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="pipeline" type="workflow"
+                    remarks="The root of the leaf name of the WDL file.
+                    Compose this value with the 'id'
+                    to get the 'table' column.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="project" type="text"
+                    remarks="Something to identify the project.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="release" type="text"
+                    remarks="The release tag of the WDL
+                    for workflows in this workload
+                    or the Git commit hash of the .wdl file
+                    if this WDL is not officially released.">
+                <constraints nullable="false"/>
+            </column>
+            <column name="started" type="timestamptz"
```

Comment:
Could this be "start_time" instead?

Reply:
A column is more than a name. It also has a type.